packages/vscode/scripts/build-browser.js

```diff
@@ -36,6 +36,7 @@ require('esbuild').build({
 	platform: 'browser',
 	format: 'iife',
 	tsconfig: './tsconfig.json',
+	external: ['synckit'],
```

Comment:
`synckit` can't run in the browser! Haven't tried with polyfills, but I doubt it will work even then. Something to investigate when we'll add support for TypeScript in the browser

Reply:
can the browser just use async / shared webworkers instead?